src/layman/map/util.py

```diff
@@ -308,3 +309,14 @@ def get_map_file_json(username, mapname):
         map_json['user'] = get_map_owner_info(username)
         map_json.pop("groups", None)
     return map_json
+
+
+def find_maps_by_grep(regexp):
+    data_dir = settings.LAYMAN_DATA_DIR
+    grep_cmd = f"grep -r -E '{regexp}' {data_dir}"
```

Comment:
Change to `f"grep -r -E '{regexp}' {data_dir}  || [[ $? == 1 ]]"`, see https://unix.stackexchange.com/a/427598 for explanation

This hopefully fixes failed test https://github.com/jirik/layman/runs/2584926010?check_suite_focus=true